Helper/Requests.php

```diff
@@ -301,20 +301,23 @@ public function buildAddressData($billingAddress, $shippingAddress, $storeId, $r
     }
 
     /**
-     * @param array $request
      * @param $amount
      * @param $currencyCode
      * @param $reference
+     * @param $shopperConversionId
+     * @param array $request
      * @return array
      */
-    public function buildPaymentData($amount, $currencyCode, $reference, array $request = [])
+    public function buildPaymentData($amount, $currencyCode, $reference, $shopperConversionId,array $request = []): array
```

Comment:
Changing the signature of a public method is a breaking change. Could you please pass the `$shopperConversionId` as the latest optional argument? 

Asides this matter, have you considered generating a new data builder only for shopper conversion ID? So that, you don't need to modify these mature classes.